.github/workflows/cli-action.yml

```diff
@@ -0,0 +1,26 @@
+name: Test CLI action
+
+on: [push]
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install CLI
+      uses: dopplerhq/cli-action@v1
+    - name: Test CLI
+      run: doppler --version
+  windows:
+    runs-on: windows-latest
+    steps:
+    - name: Install CLI
+      uses: dopplerhq/cli-action@v1
+    - name: Test CLI
+      run: doppler --version
+  macOS:
+    runs-on: macos-latest
+    steps:
+    - name: Install CLI
+      uses: dopplerhq/cli-action@v1
+    - name: Test CLI
+      run: doppler --version
```

Comment:
I believe we can streamline this further with the matrix functionality.

``` yaml
jobs:
  strategy:
    matrix:
      os: ["ubuntu-latest", "windows-latest", "macos-latest"]
  runs-on: ${{ matrix.os }}
  steps:
      - name: Install CLI
        uses: dopplerhq/cli-action@v1
      - name: Test CLI "${{ matrix.os }}"
        run: doppler --version
```

Reply:
You are right, though I believe this makes for a more confusing example. I expect our users to only use one OS and thus the current example may be more helpful.

Reply:
Gotcha, yea its a more confusing example. I'll approve the PR.

Reply:
FYI we link to this action from the [cli-action](https://github.com/DopplerHQ/cli-action) README.

Reply:
Can we update the ReadMe to show the GitHub action example?

Reply:
We're following GitHub's standard action readme for that section: https://help.github.com/en/actions/building-actions/creating-a-javascript-action#creating-a-readme.

Reply:
Sorry, I mean't the ReadMe in this repo. I have just updated our docs site as well.

Reply:
I added some README.md and INSTALL.md changes to this PR a few minutes ago. You may have reviewed before I pushed them up. Does that cover what you were thinking?

Reply:
Oh perfect! Feel free to merge!!